### PR TITLE
ENH: Adds support for workshop poster sessions

### LINKED ISSUE #92: Poster Sessions
Workshops should be able to have an optional poster session.

Ticket purchasers should be able to indicate which option relating to a poster session best fits their plans.

### PR DESCRIPTION
**Pull request by thermokarst**
Fixes #92

### CODE REVIEW

**Comment by thermokarst** [comment]
@jakereps, do you want to review this? If you don't have time, no worries, just let me know.

**Comment by jakereps** [comment]
Yeah, I can check it out later today!

File: payments/models.py [hunk 1]
```diff
@@ -119,6 +119,18 @@ def get_queryset(self):
                         ))
 
 
+class PosterOption(models.Model):
+    workshop = models.ForeignKey(Workshop)
+    name = models.CharField(max_length=300)
+    sort_order = models.IntegerField()
```
Comment (jakereps) [inline payments/models.py]: Should we provide a help text for this? Had to look at the code to see what was expected when creating a test workshop.

**Comment by jakereps** [comment]
Looks good @thermokarst, just one question then this should be good.

File: payments/models.py (hunk 1 above)
Comment (thermokarst) [inline payments/models.py]: Oh good call!

**Pull request merged**